roxie/ccd/ccdserver.cpp

```diff
@@ -24226,6 +24229,16 @@ class CRoxieServerIndexCountActivity : public CRoxieServerIndexActivity
                 else
                     countHelper.onLimitExceeded();
             }
+            else if (totalCount > keyedLimit)
```

Comment:
does it matter that this is post filter rather than keyed filter?

Reply:
As discussed in Flowdock, I think not. it will match the behaviour of the output case